be/src/runtime/stream_load/stream_load_executor.cpp

```diff
@@ -111,6 +106,10 @@ Status StreamLoadExecutor::execute_plan_fragment(StreamLoadContext* ctx) {
                 ctx->write_data_cost_nanos = MonotonicNanos() - ctx->start_write_data_nanos;
                 ctx->promise.set_value(status);
 
+                if (!executor->runtime_state()->get_error_log_file_path().empty()) {
```

Comment:
why are you removing `ctx->number_filtered_rows > 0`?

Reply:
I notice that the result of `ctx->number_filtered_rows > 0` is not strictly related to `executor->runtime_state()->get_error_log_file_path()`.

Reply:
IMO the error message is only useful when there is filtered rows. When the filtered rows is 0, no need to create this URL